frontend/src/components/layout/Navigation.tsx

```diff
@@ -1,16 +1,15 @@
 import "./Navigation.css";
 import {Link, useNavigate} from "react-router-dom";
 import React, {useState} from "react";
-import {PostAdd} from "@mui/icons-material";
-import {Button, Menu, MenuItem} from "@mui/material";
+import {Menu, MenuItem} from "@mui/material";
 
 export default function Navigation() {
     const[anchorEl, setAnchorEl] = useState<null | HTMLElement>(null);
     const open = Boolean(anchorEl);
     const navigate = useNavigate();
 
 
-    const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    const handleClick = (event: React.MouseEvent<HTMLAnchorElement>) => {
```

Comment:
handleClick is saying nothing here. A better naming could be handleClickAddRecipe for example :)

Reply:
done!